src/util-napatech.c

```diff
@@ -916,16 +926,28 @@ int NapatechGetStreamConfig(NapatechStreamConfig stream_config[])
 
                     char copystr[16];
                     strlcpy(copystr, stream->val, 16);
-                    ByteExtractStringUint16(&start, 10,  0, copystr);
-                    ByteExtractStringUint16(&end, 10,  0, strchr(copystr, '-') + 1);
+                    if (StringParseUint16(&start, 10, 0, (const char *)copystr) < 0) {
```

Comment:
This may be ok --- but, Napatech stream ids are 8 bit values (max 255) so perhaps we should use `uint8`? Perhaps this is ok as `uint16_t` is used in the `NapatechStreamConfig` struct.

@pjyoung1 Any comments?

Reply:
I often like to know when a value is out of range, vs failed to parse. Can lead to much better error messages, but does separate the parseing and the range checking adding more code.

Reply:
I agree that this would be cleaner as uint8 instead of uint16.  (It could probably be changed to uint8_t in the NapatechStreamConfig as well.)

Reply:
> I often like to know when a value is out of range, vs failed to parse

That could be implemented by functions such as `StringParseU64RangeCheck` returning different error codes